.orchestra/support/mass-testing/download_binaries

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
```

Comment:
Always use `kebab-case` for executables. `snake_case` is for Python files when in a module.
Fix for all executables in this PR.

Reply:
Changed